jwst/dark_current/dark_sub.py

```diff
@@ -87,6 +87,13 @@ def do_correction(input_model, dark_model, dark_output=None):
         # They match, so we can subtract the dark ref file data directly
         output_model = subtract_dark(input_model, dark_model)
 
+        # If the user requested to have the dark file saved,
+        # save the reference model as this file. This will
+        # ensure consistency fro the user's standpoint
+        if dark_output is not None:
+            log.info('Writing dark reference model to %s', dark_output)
```

Comment:
Your typical Joe User might not fully appreciate the meaning of "reference model". Perhaps something a little simpler, like "dark current data"? We have to be careful with the phrasing so that we don't give the impression that it's just a copy of the reference file - it is in some cases, when no averaging is done, but the whole point of having this option is to get a copy of it when it's not the same as the reference file (i.e. when it's been averaged). So I think something like "dark current data" is generic enough to not imply it's just another copy of the reference file.